webpack.prod.config.js

```diff
@@ -1,9 +1,19 @@
+const fs = require('fs');
+const dotenv = require('dotenv');
 const path = require('path');
 const { createConfig } = require('@edx/frontend-build');
 const CopyPlugin = require('copy-webpack-plugin');
 
 const config = createConfig('webpack-prod');
 
+// Overides development variables, this is required because this line
+// https://github.com/nelc/frontend-build/blob/open-release/palm.nelp/lib/plugins/paragon-webpack-plugin/ParagonWebpackPlugin.js#L20-L22
+// is always executed no matter the environment
```

Comment:
```suggestion
// is always executed no matter the environment ?
```
Is using webpack-prod config file so I i think is prod env, or what do you think of env?